apisix/plugins/kafka-logger.lua

```diff
@@ -82,6 +84,11 @@ local function send_kafka_data(conf, log_message)
     end
 
     broker_config["request_timeout"] = conf.timeout * 1000
+    broker_config["partitioner"] = function(key, num, correlation_id)
```

Comment:
It will be better if you can add test cases to cover.

Reply:
we can use "TEST 4" and "TEST 5" of kafka-logger.t test it，and number of partitions for a kafka topic >= 2.

Reply:
> It will be better if you can add test cases to cover.

we can use "TEST 4" and "TEST 5" of kafka-logger.t test it，and number of partitions for a kafka topic >= 2.